CMakeLists.txt

```diff
@@ -16,76 +16,147 @@ find_package(catkin REQUIRED COMPONENTS
   rviz_visual_tools
   tf_conversions
   interactive_markers
+  rviz
+  std_msgs
+  geometry_msgs
 )
 
-find_package(Eigen REQUIRED)
+find_package(Eigen3 REQUIRED)
+
+# Eigen 3.2 (Wily) only provides EIGEN3_INCLUDE_DIR, not EIGEN3_INCLUDE_DIRS
+if(NOT EIGEN3_INCLUDE_DIRS)
+  set(EIGEN3_INCLUDE_DIRS ${EIGEN3_INCLUDE_DIR})
+endif()
+
 find_package(Boost REQUIRED)
 
 catkin_package(
   INCLUDE_DIRS
-    include
+  include
   LIBRARIES
-    ${PROJECT_NAME}_manual_tf_alignment
-    ${PROJECT_NAME}_imarker_simple
+  ${PROJECT_NAME}_gui
+#    ${PROJECT_NAME}_manual_tf_alignment
+#    ${PROJECT_NAME}_imarker_simple
```

Comment:
fyi i've moved the imarker_simple functionality into rviz_visual_tools today, so this functionality is deprecated here